TaskWithMemoryAllocation/TaskWithMemoryAllocation.cpp

```diff
@@ -0,0 +1,26 @@
+﻿#include <iostream>
+#include <string>
+
+int main(int argc, char* argv[])
+{
+	if (argc == 1)
+	{
+		std::cout << "No command line arguments " << std::endl;
+		return 1;
+	}
+	std::cout << argv[1] << std::endl;
+	size_t num = std::stoul(argv[1]);
```

Comment:
What if I pass a word to your program?

Reply:
Regarding to your question, you highly likely crashed with std::out_of_range at this point. It forced me to rewrite this line to cause an std::bad_alloc in my hardware:
`auto num = std::stoull(argv[1]);`

Reply:
please handle the exception if your input is not a number.

Reply:
Done